openslide/lowlevel.py

```diff
@@ -187,7 +247,7 @@ def _func(name, restype, argtypes, errcheck=_check_error):
 try:
     detect_vendor = _func('openslide_detect_vendor', c_char_p, [_utf8_p], _check_string)
 except AttributeError:
-    raise OpenSlideError('OpenSlide >= 3.4.0 required')
+    raise OpenSlideVersionError('3.4.0')
```

Comment:
so with the new _func implementation, this could also be written as,
`detect_vendor = _func('openslide_detect_vendor', c_char_p, [_utf8_p], _check_string, minimum_version='3.4.0')`

Reply:
`_func(..., minimum-version='...')` defers the check until the function is actually called.  3.4.0 is a hard minimum for OpenSlide Python, so in this case we want to fail when `openslide.lowlevel` is first imported.

Reply:
I didn't look at the `_func` implementation close enough, in that case it makes sense to catch it early here.